### ISSUE TITLE
can not import simtk.openmm

### ISSUE THREAD

**Issue by XiaojuanHu** [pandegroup/openmm#2123]
Hi,

when i  used  "from simtk.openmm import  *" , i got an error:

'Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/programs/openmm-master/wrappers/python/simtk/openmm/__init__.py", line 19, in <module>
    from simtk.openmm.openmm import *
ModuleNotFoundError: No module named 'simtk.openmm.openmm''

and there is no openmm.py file in the /wrappers/python/simtk/openmm folder.

if i command the line ' from simtk.openmm.openmm import *' in simtk/openmm/__init__.py, i got another error:

Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/programs/openmm-master/wrappers/python/simtk/openmm/__init__.py", line 21, in <module>
    from simtk.openmm.mtsintegrator import MTSIntegrator
  File "/programs/openmm-master/wrappers/python/simtk/openmm/mtsintegrator.py", line 35, in <module>
    from simtk.openmm import CustomIntegrator
ImportError: cannot import name 'CustomIntegrator'

what should i do to solve this problem?

Thanks!

**Comment by jchodera** [comment]
Hi! How did you install OpenMM?

**Comment by XiaojuanHu** [comment]
Thanks for the quick reply! 

i followed the chapter 'compiling openmm from source code' in openmm user guide.

**Comment by peastman** [comment]
You must have put your python code inside the OpenMM source tree, so that it's trying to import the files from there instead of the installed version.  You need to put it somewhere else.

**Comment by jchodera** [comment]
Do you absolutely need to compile from source? Can you install via conda instead?

http://docs.openmm.org/latest/userguide/application.html#installing-openmm

**Comment by XiaojuanHu** [comment]
thanks! problem solved

**Comment by jchodera** [comment]
Great! Can you close the issue if the problem is solved?

**Comment by pratapslair** [comment]
I faced the same issue. I installed OpenMM through 'pip install openmm' command. However, when i tried to run 

from simtk.openmm.app import *
from simtk.openmm import *
from simtk.unit import *
from sys import stdout

it showed the same error again _ModuleNotFoundError: No module named 'simtk.openmm'_. So i just did Import simtk and proceed to run. This time it worked. Can you let me know if there is any update to the syntax in the latest version that's causing the error or is it entirely different?

**Comment by peastman** [comment]
OpenMM is distributed through conda, not pip.  See the installation instructions at http://docs.openmm.org/latest/userguide/application.html#installing-openmm.

**Comment by jchodera** [comment]
That reminds me that we have to do something about the PyPI `openmm` package.
Opened a new issue here: https://github.com/openmm/openmm/issues/3131

**Comment by AhmetGG** [comment]
Hi,
I installed OpenMM via conda. I use jupyter notebook but when I use "from simtk.openmm import *" , i got an error:
ModuleNotFoundError: No module named 'simtk''

I installed it in this way : 
conda create -n openmm-env -c conda-forge openmm
conda activate openmm-env

There is  openmm ın a env file of anaconda file. How can I import this sim tk model. I would greatly appreciate your kind help.Thanks
